custom_dc/cloudsql_env.list

```diff
@@ -1,12 +1,24 @@
+# (Optional) Secret token to authorize users to perform /admin page operations.
+ADMIN_SECRET=
+# The name of the Cloud SQL instance, in the form <PROJECT_ID>:<LOCATION>:<INSTANCE_ID>.     
+CLOUDSQL_INSTANCE=
+# API key for accessing the base Data Commons API.
+DC_API_KEY=
```

Comment:
Can we do logical grouping of the vars? e.g.

* Keys: DC_API_KEY, MAPS_API_KEY
* DB: USE_CLOUD_SQL, CLOUDSQL_INSTANCE, DB_NAME, DB_USER, DB_PASS
* NL: ENABLE_MODEL
* Others: FLASK_ENV, ADMIN_SECRET, REDIS_HOST

Similarly for the other file.